src/application/components/JsonDoc/Documentation.js

```diff
@@ -28,25 +28,23 @@ class Documentation extends Component {
   constructor(props: Props) {
     super(props);
     this.state = { height: 0 };
-    this.updateContainerHeight = this.updateContainerHeight.bind(this);
   }
 
   componentDidUpdate() {
     this.updateContainerHeight();
   }
 
   container: HTMLDivElement;
-  updateContainerHeight: () => void;
 
-  updateContainerHeight() {
+  updateContainerHeight: () => void = () => {
```

Comment:
Should be able to write this as 

`updateContainerHeight = (): void => {`

Which is a bit easier to read and probably more consistent with how we write this.